docs/storybook/stories/StorybookComponents/ColorScale/ColorScale.tsx

```diff
@@ -24,7 +25,11 @@ export const ColorScale = ({color}: ColorScaleProps) => {
   return (
     <div
       className="ColorScale--block"
-      style={{backgroundColor: `var(--${color})`, color: textColor}}
+      style={{
+        backgroundColor: `var(--${color})`,
+        color: textColor,
+        border: border ? '1px solid var(--color-scale-gray-1)' : 'none',
```

Comment:
```suggestion
        border: border ? '1px solid var(--borderColor-default)' : 'none',
```

What do you think of using the default border color?